backend/entities/Project.cs

```diff
@@ -118,6 +122,8 @@ public class Project : BaseEntity
         /// <summary>
         /// get/set - When the project was externally marketed.
         /// </summary>
+        [NotMapped]
+        [JsonProperty]
```

Comment:
I'd extract all these extra properties and create a `DisposalProject` in the `/Models` namespace within this library.

Reply:
If we go that route I think I'd need to update the Project Service to operate on the new DisposalProject instead of the Project object to keep the mappings working the way they do now. Alternatively I could map the model to the Project, and then convert the project to a DisposalProject by deserializing the metadata right after it was serialized. I'm not a fan of either of those options, but maybe I don't understand the suggestion.

Reply:
You've essentially lost some of the reason to use the `metadata` property by keeping `NotMapped` properties on the entity `Project`.  It makes more sense to have a specific model for each given project type.  Thus creating a `DisposalProject` model that has these properties which can be used to work with the `Metadata` properties.

Currently yes it would require updating the `ProjectService` functions, but could theoretically be handled via generics.

Reply:
maybe a partial would work here?

Reply:
A temporary option, but in truth isn't any different.

Reply:
I think I'll hold off on this for now since this at least alleviates some of the pain in the database. If I have time I'll see if I can enhance this further.